tests/functional/test_fusion_ra.py

```diff
@@ -200,7 +211,7 @@ def test_ra_user_does_not_exist(
     with pytest.raises(AnsibleFailJson):
         fusion_ra.main()
 
-    ra_mock.list_role_assignments.assert_not_called()
+    ra_mock.list_role_assignments.assert_called_with(role_name=module_args["role"])
```

Comment:
If parameters are wrong `list_role_assignments` shouldn't be called at all. Do the test pass?

Reply:
fixed in [d9f52e1](https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/135/commits/d9f52e1f74613e780884dbedee1eac16edea0e29) and [84a7043](https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/135/commits/84a70432496f90bad925c8f4c14bc777a7aaa399)